doc/source/serve/deployment-graph.md

```diff
@@ -394,31 +420,21 @@ more info on these options.
 
 Now we're done! The full example below covers the full example for you to try out.
 
-```{code-cell} ipython3
-:tags: [remove-cell]
```

Comment:
we should show some of these so when people copy the full example from the web page, the code works end to end